### PR TITLE
Remove entries that are handled by GOG manifest

### PR DESCRIPTION
**Pull request by imLinguin**
Removed entries that are already handled by GOG integration. In order to avoid redundancy

### CODE REVIEW

File: 1123662938-gog.json
Comment (arielj) [inline 1123662938-gog.json:1]: I can confirm this one is not needed anymore

File: 1189268228-gog.json
Comment (arielj) [inline 1189268228-gog.json:1]: I can confirm this one is not needed anymore

File: 1193585573-gog.json
Comment (arielj) [inline 1193585573-gog.json:1]: I don't own this one in GOG so I don't know, I only have it on Epic

File: 1207658859-gog.json
Comment (arielj) [inline 1207658859-gog.json:1]: This works, but there's one difference: physx opens an installer and the user has to follow the steps to install it. With winetricks we could avoid that (now winetricks happen after the setup anyway so the physx installer/wizard is displayed anyway.

I think it's fine to remove this but I wonder if we can force some argument if we detect we are installing physx to make it install in the background like winetricks does?

File: 1207660473-gog.json
Comment (arielj) [inline 1207660473-gog.json:1]: Works fine without this :+1:

File: 1482504285-gog.json
Comment (arielj) [inline 1482504285-gog.json:1]: I can't test this one, I own the game but in epic

File: 1345182195-gog.json
Comment (arielj) [inline 1345182195-gog.json:1]: This game works fine without this

interestingly, this one installs physx like empire earth 3, but it requires no user interaction, I think the difference is this installs phsyx and empire earth installs physx and physx legacy and the legacy one needs user interaction

File: 1425039730-gog.json
Comment (arielj) [inline 1425039730-gog.json:1]: game works fine without this

File: 1731318270-gog.json
Comment (arielj) [inline 1731318270-gog.json:1]: This game works fine without this and I think it's the issue/fix for lego batman 2 (I don't own that game on GOG but I added that from some discord thread I read, and if I remember right it was the same issue)

File: 1914219141-gog.json
Comment (arielj) [inline 1914219141-gog.json:1]: works fine without this :+1:

File: 1207658859-gog.json
Comment (imLinguin) [reply 1207658859-gog.json:1]: Well, we install physx in the background, what we are also doing is installing legacy package and that one doesn't seem to respect any silent flags

File: 1207658930-gog.json
Comment (arielj) [inline 1207658930-gog.json:1]: works fine without this

**Review by arielj — APPROVED** [review]
Looks good to me. There are 2 games I can't test: lego batman, and batman arkham asylum, I own those games but in Epic, not in GOG (I added them to the repo based solutions found in discord during support)

The lego batman I imagine it's fine to remove, I remember it was the same description of the problem as lego star wars.

for the other batman game, I imagine it's fine to remove it, I just don't own the game to confirm 100%

**Comment by imLinguin** [comment]
I assume it's safe as DirectX redist covers d3dcompilers

**Pull request merged**